src/test/java/com/cmlteam/util/UtilTest.java

```diff
@@ -0,0 +1,109 @@
+package com.cmlteam.util;
+
+import junit.framework.Assert;
+import junit.framework.TestCase;
```

Comment:
Please update Junit framework to latest version (5.x). It has API slightly changed, relies on annotations (like @Test) vs naming conventions.

Reply:
done